CHANGELOG.md

```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v9.2.0](https://github.com/puppetlabs/puppetlabs-postgresql/tree/v9.2.0) - 2023-08-28
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-postgresql/compare/v9.1.0...v9.2.0)
+
+### Added
+
+- Add Debian 12 compatibility [#1460](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1460) ([bastelfreak](https://github.com/bastelfreak))
+- puppet/systemd: Allow 5.x & puppetlabs/concat: Allow 9.x [#1448](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1448) ([bastelfreak](https://github.com/bastelfreak))
+- Add default version for Fedora 37, 38 [#1421](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1421) ([lweller](https://github.com/lweller))
+- Defaulting password encryption for version above 14 [#1406](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1406) ([cruelsmith](https://github.com/cruelsmith))
+
+### Changed
+- puppetlabs/stdlib: Require 9.x [#1449](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1449) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
This one is marked as backwards incompatible.

Reply:
I'm not really sure how the changelog generator works. I assumed this will pop up in it's own section like '### Breaking changes'.

@ekohl how should we proceed here? Make a major release now and another one in the next weeks with the changed type/provider + stricter $ports datatype?

Reply:
We reverted it so we can do a minor release